ONSdigital/dp-hierarchy-api
# Use latest dp-graph to sort hierarchies according to order property

### What

Upgrade dp-graph dependency to 2.11.0 to use the updated hierarchy query that sorts by order.

### How to review

- Make sure go.mod uses dp-graph version 2.11.0
- Make sure dependencies are 'tidy', you can run `go mod tidy` and validate that go.sum didn't change
- (optional) - I added the order property to some hierarchy nodes. After tunneling to Neptune cluster in develop, you can try the following request: `GET http://localhost:22600/hierarchies/76f512ee-eeb7-4b95-a8a5-b89eb48fb087/aggregate/cpih1dim1T120000
  - Expected response:
```

{
    "label": "12 Miscellaneous goods and services",
    "children": [
        {
            "label": "12.6 Financial services (nec)",
            "no_of_children": 1,
            "links": {
                "code": {
                    "id": "cpih1dim1G120600",
                    "href": "http://localhost:22400/code-lists/cpih1dim1aggid/codes/cpih1dim1G120600"
                },
                "self": {
                    "id": "cpih1dim1G120600",
                    "href": "http://localhost:22600/hierarchies/76f512ee-eeb7-4b95-a8a5-b89eb48fb087/aggregate/cpih1dim1G120600"
                }
            },
            "has_data": true
        },
        {
            "label": "12.5 Insurance",
            "no_of_children": 3,
            "links": {
                "code": {
                    "id": "cpih1dim1G120500",
                    "href": "http://localhost:22400/code-lists/cpih1dim1aggid/codes/cpih1dim1G120500"
                },
                "self": {
                    "id": "cpih1dim1G120500",
                    "href": "http://localhost:22600/hierarchies/76f512ee-eeb7-4b95-a8a5-b89eb48fb087/aggregate/cpih1dim1G120500"
                }
            },
            "has_data": true
        },
        {
            "label": "12.4 Social protection",
            "no_of_children": 1,
            "links": {
                "code": {
                    "id": "cpih1dim1G120400",
                    "href": "http://localhost:22400/code-lists/cpih1dim1aggid/codes/cpih1dim1G120400"
                },
                "self": {
                    "id": "cpih1dim1G120400",
                    "href": "http://localhost:22600/hierarchies/76f512ee-eeb7-4b95-a8a5-b89eb48fb087/aggregate/cpih1dim1G120400"
                }
            },
            "has_data": true
        },
        {
            "label": "12.3 Personal effects (nec)",
            "no_of_children": 2,
            "links": {
                "code": {
                    "id": "cpih1dim1G120300",
                    "href": "http://localhost:22400/code-lists/cpih1dim1aggid/codes/cpih1dim1G120300"
                },
                "self": {
                    "id": "cpih1dim1G120300",
                    "href": "http://localhost:22600/hierarchies/76f512ee-eeb7-4b95-a8a5-b89eb48fb087/aggregate/cpih1dim1G120300"
                }
            },
            "has_data": true
        },
        {
            "label": "12.7 Other services (nec)",
            "no_of_children": 1,
            "links": {
                "code": {
                    "id": "cpih1dim1G120700",
                    "href": "http://localhost:22400/code-lists/cpih1dim1aggid/codes/cpih1dim1G120700"
                },
                "self": {
                    "id": "cpih1dim1G120700",
                    "href": "http://localhost:22600/hierarchies/76f512ee-eeb7-4b95-a8a5-b89eb48fb087/aggregate/cpih1dim1G120700"
                }
            },
            "has_data": true
        },
        {
            "label": "12.1 Personal care",
            "no_of_children": 2,
            "links": {
                "code": {
                    "id": "cpih1dim1G120100",
                    "href": "http://localhost:22400/code-lists/cpih1dim1aggid/codes/cpih1dim1G120100"
                },
                "self": {
                    "id": "cpih1dim1G120100",
                    "href": "http://localhost:22600/hierarchies/76f512ee-eeb7-4b95-a8a5-b89eb48fb087/aggregate/cpih1dim1G120100"
                }
            },
            "has_data": true
        }
    ],
    "no_of_children": 6,
    "links": {
        "code": {
            "id": "cpih1dim1T120000",
            "href": "http://localhost:22400/code-lists/cpih1dim1aggid/codes/cpih1dim1T120000"
        },
        "self": {
            "id": "cpih1dim1T120000",
            "href": "http://localhost:22600/hierarchies/76f512ee-eeb7-4b95-a8a5-b89eb48fb087/aggregate/cpih1dim1T120000"
        }
    },
    "has_data": true,
    "breadcrumbs": [
        {
            "label": "Overall Index",
            "no_of_children": 12,
            "links": {
                "code": {
                    "id": "cpih1dim1A0",
                    "href": "http://localhost:22400/code-lists/cpih1dim1aggid/codes/cpih1dim1A0"
                },
                "self": {
                    "href": "http://localhost:22600/hierarchies/76f512ee-eeb7-4b95-a8a5-b89eb48fb087/aggregate"
                }
            },
            "has_data": true
        }
    ]
}
```

### Who can review

Anyone
